daemon/src/taker_cfd.rs

```diff
@@ -370,7 +370,10 @@ impl Actor {
             })
             .await?;
 
-        // TODO: Monitor for the transaction
+        cfd.handle(CfdStateChangeEvent::ProposalSigned(
+            TimestampedTransaction::new(tx),
+        ))?;
+        insert_new_cfd_state_by_order_id(cfd.order.id, cfd.state, &mut conn).await?;
```

Comment:
we're adding an event, that doesn't change the state - it still stays in 'Open' - and as transition timestamps can happen due to variety of reasons (main ones being settlement proposals and attestations), we can't reliably tell how long did it take since the proposal signature...